adhocracy4/maps/static/a4maps/a4maps_display_points.js

```diff
@@ -83,23 +84,31 @@ function init () {
     }
 
     function getRatings (feature) {
-      return '<span class="map-popup-upvotes">' +
-               feature.properties.positive_rating_count + ' <i class="fa fa-chevron-up" aria-hidden="true"></i>' +
+      return '<span class="map-popup-upvotes" aria-hidden="true">' +
+               feature.properties.positive_rating_count + ' <i class="fa fa-chevron-up"></i>' +
              '</span>' +
-             '<span class="map-popup-downvotes">' +
-               feature.properties.negative_rating_count + ' <i class="fa fa-chevron-down" aria-hidden="true"></i>' +
+             '<span class="map-popup-downvotes" aria-hidden="true">' +
+               feature.properties.negative_rating_count + ' <i class="fa fa-chevron-down"></i>' +
```

Comment:
also just realised this will mean that the screen reader is reading our a element which cannot be seen which is confusing for partially sighted, we should put sr content in an element that can be seen